### ISSUE TITLE
Determining vTypes of automatically created detectors in actuated traffic light

### ISSUE THREAD

**Issue by abed1856** [eclipse/sumo#5730]
Hello Jakob, (from #5125)
What should be done to affect this new optional param in tllogic?
I have used it but still sumo uses its pre-defined detectors and shows them. I also defined detectors in additional files but the the "max-gap" defined in param does not connect to custom defined detector (neither by activating "show-detectors" and "max-gap" nor by deactivating them)

    what if i just want to define these detectors for separated bicycle lanes ? (which are in parallel to major highway)

    what if you wanna define a specific vType for automatically interseted detectors (for example only bicycles)
    my definitions are as below:
    <tlLogic id="gneJ5" type="actuated" programID="0" offset="0">
        <phase duration="5" state="rGrG"/>
        <param key="file" value="NULL"/>
		<param key="detectors" value="det1"/>
        <param key="freq" value="300"/>
        <param key="max-gap" value="50.0"/>
        <param key="show-detectors" value="true"/>
        <phase duration="3"  state="ryry"/>
        <phase duration="5" state="GrGr" minDur="5" maxDur="100"/>

**Comment by namdre** [comment]
As you can see from the ticket state 'open' of #5125, and the lack of commit references in the ticket, this issue is still in a conceptual stage and not yet usable. However,  This also means, there is currently no way to use vType-specific detectors for actuated traffic lights. However, if you use separate bike lanes, they will also receive detectors and it should work as expected.

**Comment by abed1856** [comment]
Thank you very much for fast reply.

**Comment by behrisch** [comment]
duplicate of #5125
